frontend/packages/data-portal/app/utils/filter.ts

```diff
@@ -0,0 +1,42 @@
+import { isNumber } from 'lodash-es'
+
+import { Runs_Bool_Exp } from 'app/__generated__/graphql'
+import { DEFAULT_TILT_MAX, DEFAULT_TILT_MIN } from 'app/constants/tiltSeries'
+
+export function getTiltValue(value: string | null) {
+  if (value && !Number.isNaN(+value)) {
+    return +value
+  }
+
+  return null
+}
+
+export function getTiltRangeFilter(
+  initialTiltMin: string,
+  initialTiltMax: string,
+) {
+  let tiltMin = getTiltValue(initialTiltMin)
+  let tiltMax = getTiltValue(initialTiltMax)
+
+  if (isNumber(tiltMin) && !isNumber(tiltMax)) {
+    tiltMax = DEFAULT_TILT_MAX
```

Comment:
Curious on how we arrived at the `DEFAULT_TILT_MAX` and `DEFAULT_TILT_MIN` values?

Reply:
@Janeece, sorry if this was addressed elsewhere; the minimum angle values can be negative, so we are being restrictive with our default value of 0. 

What do you think about either setting min and max defaults dynamically from the DB or not adding them to the filter when not explicitly specified?

Reply:
This should be consistent with how we implemented it on the all datasets filter. How is that one working?

HOWEVER, I think its good to question this filter — its design is proving to be confusing and i think i finally understand why it doesn't line up. 

Tilt range is the angle between the min and max tilt angles (-60min tilt angle to 60 max title angle = 180° tilt range). 

The filter can either (1) let users pick a single tilt range, like 60° tilt range, or (2) let users pick a range of tilt ranges, like 20° to 180° tilt range. 

1. Would result in a narrow filter, only one tilt range some value between 0° and 180° (ex filter by 180° tilt range). The min/max filter inputs in this case would be the min and max tilt angles (could be negative, like -60° to 60°)
2. Would let users filter by a range of tilt ranges, so any range between the min and max tilt range would be ok. The min/max filter inputs would be the minimum tilt range to the maximum tilt range (must be positive, like 0 to 180°). 

My understanding is that this filter should be accomplishing (2). But the copy (and maybe the form structure) are actually describing (1)  

I need to re-confirm with which it is with Anchi/Utz to know how to correct the confusion.

Reply:
Additionally, on the single run page we present tilt range as the min to max angle, not the range total